app/js/components/manageApps/Addon.jsx

```diff
@@ -252,7 +284,8 @@ class Addon extends Component {
             app={app}
             handleDelete={this.handleDelete}
             isOpen={isOpen}
-            hideModal={this.hideModal} />
+            hideModal={this.hideModal}
+            appType={app.uuid ? app.uuid : "owa"} />
```

Comment:
`app.uuid` is not an appType and should not be treated as such. 
`appType = {app.uuid ? 'module': 'owa'}` would be a better exprerssion

Reply:
Changed the name to reflect appUuid and not appType